xml/containerized-kubernetes.xml

```diff
@@ -62,23 +60,41 @@
  </sect1>
  <sect1 xml:id="kube-prereq">
   <title>Prerequisites</title>
-
   <para>
-   Before you start deploying, you need to have:
+   The minimum requirements and prerequisites to deploy &productname; &productnumber;
+   on top of &casp; 4 &kube; cluster are as follows:
   </para>
-
   <itemizedlist>
    <listitem>
     <para>
-     A running &casp; 4 cluster.
+     A running &casp; 4 cluster. You need to have an SCC account with an
```

Comment:
Would suggest some wording similar to "... account with a SES6 subscription. You can activate a 60-day..." because non-eval subscriptions also work.

Reply:
"You can activate a 60-day free Evaluation here <link> because non-evaluation subscriptions also work."

Uhh, is that what you mean to say  @BlaineEXE ?